daemon/src/engine/client/cg_msgdef.h

```diff
@@ -719,6 +721,9 @@ using CGameTextInptEvent = IPC::SyncMessage<
 using CGameFocusEventMsg = IPC::SyncMessage<
 	IPC::Message<IPC::Id<VM::QVM, CG_FOCUS_EVENT>, bool>
 >;
+using CGameRecvMessageMsg = IPC::SyncMessage<
+	IPC::Message<IPC::Id<VM::QVM, CG_RECV_MESSAGE>, IPC::SharedMemory, size_t, int>
```

Comment:
Ah it is engine -> VM. How about creating a commandbuffer for that direction too instead of making a shmem each time?